examples/src/llms/googlevertexai-streaming.ts

```diff
@@ -1,6 +1,8 @@
-import { GoogleVertexAI } from "@langchain/community/llms/googlevertexai";
+import { VertexAILLM } from "@langchain/google-vertexai";
+// Or, if using the web entrypoint:
+// import { VertexAILLM } from "@langchain/google-vertexai-web";
 
-const model = new GoogleVertexAI({
+const model = new VertexAILLM({
```

Comment:
nit: `GoogleVertexAI` I think is a better name? We're not calling it e.g. `OpenAILLM`

Reply:
Oh, I see it's `VertexAI` in Python. That's fine, we should match.